src/node_http_parser.cc

```diff
@@ -462,19 +458,14 @@ class Parser : public AsyncWrap, public StreamListener {
     if (!cb->IsFunction())
       return 0;
 
-    // We came from consumed stream
-    if (current_buffer_.IsEmpty()) {
-      // Make sure Buffer will be in parent HandleScope
-      current_buffer_ = scope.Escape(Buffer::Copy(
-          env()->isolate(),
-          current_buffer_data_,
-          current_buffer_len_).ToLocalChecked());
-    }
+    // Make sure Buffer will be in parent HandleScope
+    Local<Object> current_buffer = scope.Escape(
```

Comment:
I don't think the escaping is necessary anymore here, since it is no longer stored anywhere. The scope can become a regular HandleScope.

Reply:
Fair point. Will remove it soon!